change_all_yo_letters_in_dictionaries.py

```diff
@@ -0,0 +1,17 @@
+import helper
+
+
+def change_all_yo_letters_in_file(file_path):
+    file_data = open(file_path, 'r+')
+    all_lines = list(map(helper.remove_new_line_symbol, file_data.readlines()))
+
+    all_lines_updated = []
+
+    for line in all_lines:
+        all_lines_updated.append(line.replace('ё', 'е'))
```

Comment:
Could be simplified a bit

```suggestion
    all_lines_updated = list(map(lambda line: line.replace('ё', 'е'), all_lines))
```